clang/lib/AST/DeclCXX.cpp

```diff
@@ -81,6 +82,7 @@ CXXRecordDecl::DefinitionData::DefinitionData(CXXRecordDecl *D)
       HasPrivateFields(false), HasProtectedFields(false),
       HasPublicFields(false), HasMutableFields(false), HasVariantMembers(false),
       HasOnlyCMembers(true), HasInitMethod(false), HasInClassInitializer(false),
+      HasUninitializedExplicitInitFields(false),
```

Comment:
THIS I think makes sense as the alternative cost is too high, despite having a cost on every `CXXRecordDecl`.

That said, should this be on `RecordDecl` instead?